sympy/integrals/tests/test_transforms.py

```diff
@@ -839,6 +856,26 @@ def simp_hyp(expr):
     assert ILT( (s * eye(2) - Matrix([[1, 0], [0, 2]])).inv(), s, t) ==\
         Matrix([[exp(t)*Heaviside(t), 0], [0, exp(2*t)*Heaviside(t)]])
 
+
+def test_laplace_inverse_laplace_transform():
+    # These tests covers inverse_laplace engine rules by calculating
+    # in both directions
+    a, b, c, d = symbols('a b c d', positive=True)
+    t = symbols('t')
+    def bothways(F):
+        f = inverse_laplace_transform(F, s, t)
+        F2 = laplace_transform(f, t, s, noconds=True)
+        d = (F-F2).simplify(doit=False)
+        return d==0
+    assert bothways(b/(s**2-a**2))
+    assert bothways(b*s/(s**2-a**2))
+    assert bothways(b/(s*(s+a)))
+    assert bothways(b*s/(s+a)**2)
```

Comment:
I think it would be better if the result of the inverse transform was explicitly listed here.

Reply:
I'll do that later, let's first finish `laplace_transform`, then I will structure `inverse_laplace_transform` in a similar fashion